### PR TITLE
[Bug]: Fix date histogram issue in BarGarph

### PR DESCRIPTION
**Pull request by nxanil**
@natabal @killanch 

Fix issue #324 and #326 to add support of date Histogram on the x-axis. Please have a look.
To apply date histogram, need to set `xTickFormat` from the configuration.

### CODE REVIEW

**Comment by killanch** [comment]
@nxanil I tested the fix, we still have one issue, ordering of time.
With the same data array example I had provided in https://github.com/nuagenetworks/vis-graphs/issues/324, the graph appears as below,
date_order

The latest date is rendered on the left side of the graph and the earliest date on the right. It should have been the other way.
`"xTickFormat": "%m/%d %H:%M"`
Could you pls have a look?

**Comment by nxanil** [comment]
> @nxanil I tested the fix, we still have one issue, ordering of time.
> With the same data array example I had provided in #324, the graph appears as below,
> date_order
> 
> The latest date is rendered on the left side of the graph and the earliest date on the right. It should have been the other way.
> `"xTickFormat": "%m/%d %H:%M"`
> Could you pls have a look?

@killanch 
The sorted issue has been fixed. Please have a look.

**Comment by killanch** [comment]
@nxanil All issues appear to be resolved,
linegraph

@natabal We could merge this PR

File: Graphs/DynamicBarGraph/index.js [hunk 1]
```diff
@@ -180,26 +182,88 @@ class BarGraph extends XYGraph {
       this.customExtent[1] = this.customExtent[1] + Math.abs(difference);
 
     return this.customExtent
-}
+  }
+
+  setAxis(data) {
```
Comment (natabal) [inline Graphs/DynamicBarGraph/index.js]: why do we duplicate this here from super class XYGraph.

**Pull request closed without merging**